src/components/HomePage.js

```diff
@@ -0,0 +1,25 @@
+import {BrowserRouter, Routes, Route, Link} from 'react-router-dom';
+
+function HomePage(){
+    return <div>
+    <div>
+    <img src="../assets/beers.png"/>
```

Comment:
Those are just string, so you had to import them at the top of the page and use them later for example:

```suggestion
    <img src={beers}/>
```